Documentation/sources/push-notification/how-to-display-iam-on-push-click.md

```diff
@@ -0,0 +1,30 @@
+# Display an in-message when a push notification is clicked
+
+## Overview
+This document describes the steps that allow a designated in-app message to be shown to an end user when they open the app through a push notification click.
+
+## Add a trigger in the In-App Message Campaign
+
+1. In the Journey Optimizer UI, select the Campaign for the in-app message that needs to be shown when a push notification is clicked.
+2. Under the **Triggers** section, click on the **Edit triggers** button. Click on **Add Condition** and select the **Manual trigger** from the event dropdown. Click on **Add Condition** again and select **Custom trait** from the trait menu. Enter `adb_iam_id` as the key for the custom trait and an id that uniquely identifies the in-app message as the value. Make a note of this id as it will be used in the next step.
```

Comment:
i wonder if we should add a note in here that it's possible to enable any in-app message for this workflow by adding this to the trigger.  meaning, the condition can "existing trigger to show the message" OR "push to in-app stuff matches".

wdyt?